src/Apprenticeships/SFA.DAS.Apprenticeships.Api/Controllers/ApprenticeshipController.cs

```diff
@@ -68,8 +68,7 @@ public async Task<ActionResult> GetApprenticeshipKey(string apprenticeshipHashed
         {
             await _apiClient.PostWithResponseCode<object>(new PostCreateApprenticeshipPriceChangeRequest(
                 apprenticeshipKey,
-                request.ProviderId,
-                request.EmployerId,
+                request.Requester,
```

Comment:
In the method CreateApprenticeshipPriceChange, an Ok result is returned even if the innerApi fails, I know this is the existing behavior for the provider initiated path, but this could do with returning a negative status code on failure.

Reply:
sorted